onset_maproom/maproom.py

```diff
@@ -255,6 +257,18 @@ def map_click(click_lat_lng):
     ), round(lat_lng[0],4), round(lat_lng[1],4)
 
 
+@APP.callback(
+    Output("hover_feature_label", "children"),
+    Input({"type": "borders_adm", "index": ALL}, "hover_feature")
+)
+def write_hover_adm_label(adm_loc):
+    location_description = "the map will return location name"
+    for i, adm in enumerate(adm_loc):
+        if adm is not None:
```

Comment:
Why would it be `None`?

Reply:
if unchecked, hover_feature returns None